recipe/meta.yaml

```diff
@@ -58,7 +57,11 @@ requirements:
 
 test:
   requires:
+    # TODO: Remove the following conditional.
+    #       ref: https://github.com/numba/numba/pull/5029
+    {% if environ['PKG_VERSION'] != "0.47.0" or environ['PY_VER'] != "3.8" %}
```

Comment:
The easiest way to skip the failing annotation tests is just to not install `jinja2`.
In anticipation that the Numba folks will fix it in a future release, this conditional will pass after the next version change, to make sure we rerun those tests again. (It should be removed nonetheless as it's clutter.)